### ISSUE TITLE
Identify Shared Hits

### ISSUE THREAD

**Issue by pbutti** [LDMXAnalysis/ldmx-sw#1331]
In order to reduce the combinatorics in the track finding an remove duplicated seeds I would like to implement a way to compute the number of shared hits on each ldmx::Track. However measurements now to not have a unique identifier. One possibility is to add an identifier to each measurement in the ldmx::Measurement class, that is assigned from the Digitization Processor.
@omar-moreno

**Comment by tvami** [comment]
@EBerzin your ambiguity resolver makes this issue ready to close, right?

**Comment by EBerzin** [comment]
Yes, shared hits are now found after track finding in the CKF processor.
